src/ui/control/fullscreen_control.js

```diff
@@ -12,7 +12,7 @@ type Options = {
 };
 
 /**
- * A `FullscreenControl` control contains a button for toggling the map in and out of fullscreen mode.
+ * A `FullscreenControl` control contains a button for toggling the map in and out of fullscreen mode. `FullScreenControl` is [not currently compatible](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen#browser_compatibility) with the mobile Safari browser.
```

Comment:
Im hesitant to bake in this type of compatibility statement into our code and docs. The `container` doc on line 19 side steps it by pointing to  compatibility table as well: Maybe the same could be done here:

```suggestion
 * A `FullscreenControl` control contains a button for toggling the map in and out of fullscreen mode. See the `requestFullScreen` [compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen#browser_compatibility) for supported browsers.
```

Reply:
Great suggestion, @asheemmamoowala!